packages/documentation/README.md

```diff
@@ -40,6 +36,22 @@ _Have another more specific idea? You may want to check out our vibrant collecti
 
     Open the `my-default-starter` directory in your code editor of choice and edit `src/pages/index.js`. Save your changes and the browser will update in real time!
 
+## Pushing changes to github io
+
+1. **Create a Github personal access token**
+
+   See instructions above
+
+1. **Run build and publish task**
+
+    Navigate to the documentation package and run.
+
+    ```
+    GITHUB_API_KEY=your-personal-access-token yarn build
```

Comment:
are we using yarn or npm in this project?

Reply:
Yarn